contributor-docs/adrs/adr-021-component-adoption.md

```diff
@@ -0,0 +1,54 @@
+# Component adoption
+
+## Status
+
+| Stage    | Status |
+| -------- | ------ |
+| Approved | ✅     |
+| Adopted  | ✅     |
+
+## Context
+
+When there is a new component added to the library and it is ready for use, we need a clear path for encouraging the adoption of the new component and discourage the use of the old component, if exists.
+
+For example:
+
+```jsx
+// Tooltip v1 is discouraged
+import {Tooltip} from '@primer/react'
+
+render(
+  <Tooltip aria-label="This cannot be undone">
+    <Button>Delete</Button>
+  </Tooltip>,
+)
+```
+
+```jsx
+// Tooltip v2 is the recommended component
+import {Tooltip} from '@primer/react/next'
+
+render(
+  <Tooltip text="This cannot be undone">
+    <Button>Delete</Button>
+  </Tooltip>,
+)
+```
+
+## Decision
+
+1. Announcement on Primer changelog.
+2. Reach out to teams directly, specifically strategical services.
```

Comment:
```suggestion
2. Reach out to teams directly. It may be beneficial to prioritize communication with [strategic services](https://github.com/github/accessibility-governance/tree/main/dashboard#strategic-services) (GitHub staff only).
```